subjects/check_user_name/README.md

```diff
@@ -0,0 +1,63 @@
+## check_user_name
+
+### Instructions
+
+Sometimes it is more desirable to catch the failure of some parts of a program instead of just calling panic.
+
+For this exercise you will have to create a tool that manages users' access levels.
+
+You will have to create an `AccessLevel` enum wich could be `Guest`, `Normal`, `Admin`.
+
+You will also have to create a `User` struct which has:
+
+- elements:
+  - `name`: `String`
+  - `acessLevel`: `enum`
+- associated functions:
+  - `new`: which initializes the struct.
+  - `send_name`: which takes only **self** as argument and returns an **Option<&str>** with `None` if the user is a guest or the `name` if the `AccessLevel` has any of the other options.
+  - `check_user_name`: which takes a `User`, calls `send_name` and returns a **tuple** with `true` and the user `name` if `send_name` returns the name or `false` and `"Error: User is guest"` if not
+
+### Expected Functions and Data Structures
+
+```rust
+pub enum AccessLevel {}
+
+pub struct User {}
+
+impl User {
+  pub fn new(name: String, level: AccessLevel) -> User {}
+  pub fn send_name(&self) -> Option<&str> {}
+}
+
+pub fn check_user_name(user: &User) -> (bool, &str) {}
+```
+
+### Usage
+
+Here is a program to test your function
```

Comment:
Here unfortunately there is not a unique and standard way this sentence is ended in the rest of the exercises.
Still for new ones and when I modify the old ones I try to enforce two dots at the of the sence ( "... function:") which seems to me the most correct way to end it.

Reply:
I agree. This one slipped through when using the old subject as template. Thanks.

Reply:
Done.